Resources/Prototypes/_LostParadise/Entities/Objects/Devices/pda.yml

```
@@ -37,17 +37,23 @@
-#     id: LPPAdjutantIDCard
-#     state: pda-clear
-#   - type: Icon
-#     state: pda-clear
+- type: entity
+  parent: BasePDA
+  id: LPPAdjutantPDA
+  name: adjutant PDA
+  description: Adjutant PDA.
+  components:
+  - type: Pda
+    id: AdminAssistantIDCard # LPPAdjutantIDCard
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)